src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -230,11 +230,6 @@ public enum JvmSettings {
     OIDC_AUTH_SERVER_URL(SCOPE_OIDC, "auth-server-url"),
     OIDC_CLIENT_ID(SCOPE_OIDC, "client-id"),
     OIDC_CLIENT_SECRET(SCOPE_OIDC, "client-secret"),
-    SCOPE_OIDC_PKCE(SCOPE_OIDC, "pkce"),
-    OIDC_PKCE_ENABLED(SCOPE_OIDC_PKCE, "enabled"),
-    OIDC_PKCE_METHOD(SCOPE_OIDC_PKCE, "method"),
-    OIDC_PKCE_CACHE_MAXSIZE(SCOPE_OIDC_PKCE, "max-cache-size"),
-    OIDC_PKCE_CACHE_MAXAGE(SCOPE_OIDC_PKCE, "max-cache-age"),
```

Comment:
I may have mentioned this elsewhere but when we remove settings we should mention them in the release note snippet.